heartbeat/pgsql

```diff
@@ -1040,7 +1040,7 @@ pgsql_replication_monitor() {
 
     # I can't get master node name from $OCF_RESKEY_CRM_meta_notify_master_uname on monitor,
     # so I will get master node name using crm_mon -n
-    print_crm_mon | tr -d "\t" | tr -d " " | grep -q -e "^${RESOURCE_NAME}[(:].*[):].*Master" -e "^\*${RESOURCE_NAME}[(:].*[):].*Master"
+    print_crm_mon | grep -q -E "<resource id=\"${RESOURCE_NAME}\" .* role=\"Promoted|Master\""
```

Comment:
Use -i, I'll probably use lowercase when it changes to "promoted"

Reply:
Updated to use -i.